content/rs/references/compatibility/resp.md

```diff
@@ -0,0 +1,82 @@
+---
+Title: RESP compatibility with Redis Enterprise
+linkTitle: RESP
+description: Redis Enterprise supports RESP2 and RESP3. 
+weight: 80
+alwaysopen: false
+toc: "true"
+categories: ["RS"]
+aliases: 
+---
+
+RESP (REdis Serialization Protocol) is the protocol that clients use to communicate with Redis databases. See the [RESP protocol specification](https://redis.io/docs/reference/protocol-spec/) for more information.
+
+## Supported RESP versions
+
+- RESP2 is supported by Redis Enterprise 6.0 and later.
+
+- RESP3 is supported by Redis Enterprise 7.2 and later.
+
+## Enable RESP3
+
+To use RESP3 with Redis Enterprise:
+
+1. Upgrade Redis servers to version 7.2 or later.
+
+    For Active-Active and Replica Of databases:
+ 
+    1. Upgrade all participating clusters to Redis Enterprise version 7.2.x or later.
+ 
+    1. Upgrade all databases to version 7.x or later.
+
+1. Enable RESP3 support for your database (`enabled` by default):
+
+    ```sh
+    rladmin tune db db:<ID> resp3 enabled
+    ```
+
+ ## Deactivate RESP3
+
+ To deactivate RESP3 support for a database:
+
+ ```sh
+ rladmin tune db db:<ID> resp3 disabled
+ ```
+
+{{<note>}}
+You cannot use sharded pub/sub if you deactivate RESP3 support.
+{{</note>}}
+
+## Client prerequisites for Redis 7.2 upgrade
+
+The Redis clients [Go-Redis](https://redis.uptrace.dev/) version 9 and [Lettuce](https://lettuce.io/) versions 6 and later use RESP3 by default. If you use either client to run Redis Stack commands, you should set the client's protocol version to RESP2 before upgrading your database to Redis version 7.2 to prevent potential downtime.
```

Comment:
> to prevent potential downtime.

This needs to be clarified. Is it about downtime of RE or Application?
I suggest replacing it with something like:

_to prevent malfunction of the application due to RESP3 breaking changes_

Reply:
Updated